scripts/check-third-party-licenses.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Should this be consistent with our other SOCI scripts? i.e. should the copyright be to Soci Snapshotter Authors instead of Amazon?